plugin/PDFView.qml

```diff
@@ -132,6 +136,35 @@ SilicaFlickable {
         linkColor: Theme.highlightColor;
         pagePlaceholderColor: Theme.highlightColor;
 
+        onPageLayoutChanged: if (rememberPositionConfig.value) restorePageSettings()
```

Comment:
What confused me was layout() called in multiple places, first happening in geometry changed handler, but looking closer it seems that call to layout() just returns on no page sizes existing. Requests sizes, but I'd assume that gets ignored due to document not loaded.